packages/config/config/devices/0x019a/11_02_011.json

```diff
@@ -148,10 +240,34 @@
 				}
 			]
 		},
+		{
+			"#": "9",
+			"$if": "firmwareVersion >= 0.3",
+			"label": "Ambient light reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Report when levels in parameter 10 & 11 are passed",
+					"value": 2
+				}
+			]
+		},
 		{
 			"#": "10",
+			"$if": "firmwareVersion <= 0.2",
 			"label": "Leakage alarm",
-			"description": "0:Off 1:On",
```

Comment:
We should also normalize Off/On to Disable/Enable where it makes sense

Reply:
I agree, but I wasn't trying to do line-level edits to these as I was merging them. I've fixed this one.